src/rpc/server.cpp

```diff
@@ -493,7 +493,12 @@ UniValue CRPCTable::execute(const JSONRPCRequest &request) const
     const CRPCCommand *pcmd = tableRPC[request.strMethod];
     if (!pcmd)
         throw JSONRPCError(RPC_METHOD_NOT_FOUND, "Method not found");
-
+    if (request.URI != "" && request.URI != "/") {
+        // enforce correct endpoint usage
```

Comment:
They are in this PR because I though we have decided to not support node command on `/v1/wallet`.